### DISCUSSION TITLE
CORTX services in a VM vapp can survive reboot?

### DISCUSSION THREAD

**Discussion by korenlev** [Seagate/cortx#425] [Q&A] [answered]
seems services are not coming back up after reboot 
expected ?

**Comment by johnbent** [comment, marked as answer]
@justinzw , should/can this be converted into an Issue?

@korenlev , other people have been reporting this.  We are working to get a new OVA image released that fixes this problem.  Thanks for reporting this!

**Reply by korenlev** [reply]
hi,

suggested  fix ? - see below:

cat /etc/systemd/system/cortx_start.service

[Unit]
Description=cortx hctl start
After=network.target

[Service]
Type=idle
RemainAfterExit=yes
ExecStart=/var/tmp/cortx_start
TimeoutStartSec=30

[Install]
WantedBy=default.target

cat /var/tmp/cortx_start

#!/bin/bash
hctl start
sleep 10s
/var/tmp/services-run-cmd start

cat /var/tmp/services-run-cmd

systemctl $1 rabbitmq-server
systemctl $1 elasticsearch
systemctl $1 haproxy
systemctl $1 s3authserver
systemctl $1 sspl-ll
systemctl $1 csm_agent
systemctl $1 csm_web
suggested workaround (using systemd) provided here :
https://github.com/Seagate/cortx/issues/415
